src/core_plugins/interpreter/server/routes/socket.js

```diff
@@ -1,20 +1,34 @@
 /*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
  */
 
 import socket from 'socket.io';
-import { createHandlers } from '../lib/create_handlers';
-import { socketInterpreterProvider } from '../../common/interpreter/socket_interpret';
-import { serializeProvider } from '../../common/lib/serialize';
-import { functionsRegistry } from '../../common/lib/functions_registry';
-import { typesRegistry } from '../../common/lib/types_registry';
+import { createHandlers } from '@kbn/interpreter-utils/server/create_handlers';
+import { socketInterpreterProvider } from '@kbn/interpreter-utils/common/interpreter/socket_interpret';
+import { serializeProvider } from '@kbn/interpreter-utils/common/lib/serialize';
+import { functionsRegistry } from '@kbn/interpreter-utils/common/lib/functions_registry';
+import { typesRegistry } from '@kbn/interpreter-utils/common/lib/types_registry';
+import { getAuthHeader } from '../lib/get_auth_header';
 import { loadServerPlugins } from '../lib/load_server_plugins';
-import { getAuthHeader } from './get_auth/get_auth_header';
 
 export function socketApi(server) {
   const io = socket(server.listener, { path: '/socket.io' });
+  loadServerPlugins();
```

Comment:
Why did this change? This will likely cause possible breakage. 

What else changed? This is one of the few diffable files in this pull, so I'm concerned there are other logic changes in this pull.

Reply:
i will point out all logic changes

Reply:
i should probably even remove it from here, the functions should be registered at this time, so no need to call this function everytime someone calls getFunctionList on the